sql-parser/src/main/java/io/crate/sql/parser/Statement.g

```diff
@@ -659,35 +659,31 @@ explainOption
     ;
 
 showTablesStmt
-    : SHOW TABLES from=showTablesFrom? like=showTablesLike? -> ^(SHOW_TABLES $from? $like?)
+    : SHOW TABLES (( FROM | IN ) qname)? likeOrWhere? -> ^(SHOW_TABLES qname? likeOrWhere?)
```

Comment:
``(( FROM | IN ) qname)?`` can be unified with ``showColumnsFrom`` rule (then rename)